data/json/game_balance.json

```diff
@@ -187,5 +187,12 @@
     "info": "A boolean specifically for Crazy Cataclysm.",
     "stype": "bool",
     "value": false
+  },
+  {
+    "type": "EXTERNAL_OPTION",
+    "name": "STK",
```

Comment:
There exists no limit on the name of options. Please use a meaningful name.

Reply:
no problem